packages/react-core/src/deprecated/components/Wizard/examples/WizardWithDrawer.tsx

```diff
@@ -7,7 +7,7 @@ import {
   DrawerHead,
   DrawerPanelContent
 } from '@patternfly/react-core';
-import { Wizard } from '@patternfly/react-core/deprecated';
+import { Wizard as Wz } from '@patternfly/react-core/dist/esm/deprecated';
```

Comment:
Had to alias and update the import path here, but aliasing and updating the path in some of the other examples caused those examples to stop functioning for some reason.

Reply:
Not sure what I did differently, but I just took another swing at making the same update in the other examples and it seems to be working now? The change to import from dist and alias deprecated component names in all examples is in now.